ably-ios/ARTPresenceMessage.m

```diff
@@ -34,4 +34,8 @@ - (NSString *)description {
     return description;
 }
 
+- (NSString *)memberKey {
+    return [NSString stringWithFormat:@"%@:%@", self.connectionId, self.clientId];
```

Comment:
Should it return an empty string if the `connectionId` or `clientId` are nil?
It will return `(null):(null)` if both properties are nil.